spec/plans/import.fmf

```diff
@@ -91,8 +91,33 @@ description: |
     Inheriting or overriding remote plan config with local plan
     steps might be possible in the future but currently is not
     supported. The imported plan can be modified in only two ways.
```

Comment:
I guess the `... might be possible in the future but currently is not supported` part should be dropped, as the future apparently came now ;-)